README.md

```diff
@@ -1,26 +1,53 @@
 # Distributed
 
-The `Distributed` package provides functionality for creating and controlling multiple Julia processes remotely, and for performing distributed and parallel computing. It uses network sockets or other supported interfaces to communicate between Julia processes, and relies on Julia's `Serialization` stdlib package to transform Julia objects into a format that can be transferred between processes efficiently. It provides a full set of utilities to create and destroy new Julia processes and add them to a "cluster" (a collection of Julia processes connected together), as well as functions to perform Remote Procedure Calls (RPC) between the processes within a cluster. See the `API` section for details.
+The `Distributed` package provides functionality for creating and controlling
+multiple Julia processes remotely, and for performing distributed and parallel
+computing. It uses network sockets or other supported interfaces to communicate
+between Julia processes, and relies on Julia's `Serialization` stdlib package to
+transform Julia objects into a format that can be transferred between processes
+efficiently. It provides a full set of utilities to create and destroy new Julia
+processes and add them to a "cluster" (a collection of Julia processes connected
+together), as well as functions to perform Remote Procedure Calls (RPC) between
+the processes within a cluster. See the `API` section for details.
 
 This package ships as part of the Julia stdlib.
 
 ## Using development versions of this package
 
-To use a newer version of this package, you need to build Julia from scratch. The build process is the same as any other build except that you need to change the commit used in `stdlib/Distributed.version`.
-
-It's also possible to load a development version of the package using [the trick used in the Section named "Using the development version of Pkg.jl" in the `Pkg.jl` repo](https://github.com/JuliaLang/Pkg.jl#using-the-development-version-of-pkgjl), but the capabilities are limited as all other packages will depend on the stdlib version of the package and will not work with the modified package.
+### On Julia 1.11+
+In Julia 1.11 Distributed was excised from the default system image and became
+more of an independent package. As such, to use a different version it's enough
+to just `dev` it explicitly:
+```julia-repl
+pkg> dev https://github.com/JuliaLang/Distributed.jl.git
```

Comment:
Isn't it just `dev Distributed.jl` or is the issue that it is not registered in the registry?

Reply:
Yep, it's not registered:
```julia-repl
(@v1.11) pkg> dev Distributed.jl
    Updating registry at `~/.julia/registries/General.toml`
ERROR: The following package names could not be resolved:
 * Distributed (8ba89e20-285c-5b6f-9357-94700520ee1b in manifest but not in project)
```

Can we register it?

Reply:
Going off [this checklist](https://github.com/JuliaLang/julia/issues/50697) I've opened two PRs that should complete excision and make Distributed upgradeable:
- https://github.com/JuliaLang/Pkg.jl/pull/4111
- https://github.com/JuliaRegistries/General/pull/121256